Modules/Loadable/Segmentations/EditorEffects/qSlicerSegmentEditorAbstractEffect.cxx

```diff
@@ -142,6 +142,31 @@ void qSlicerSegmentEditorAbstractEffect::setName(QString name)
   qCritical() << Q_FUNC_INFO << ": Cannot set effect name by method, only in constructor!";
 }
 
+//-----------------------------------------------------------------------------
+QString qSlicerSegmentEditorAbstractEffect::title()const
+{
+  if (!this->m_Title.isEmpty())
+    {
+    return this->m_Title;
+    }
+  else if (!this->m_Name.isEmpty())
+    {
+    return this->m_Name;
+    }
+  else
+    {
+    qCritical() << Q_FUNC_INFO << ": Empty effect title!";
+    return QString();
+    }
+}
+
+//-----------------------------------------------------------------------------
+void qSlicerSegmentEditorAbstractEffect::setTitle(QString title)
+{
+  Q_UNUSED(title);
+  qCritical() << Q_FUNC_INFO << ": Cannot set effect title by method, only in constructor!";
```

Comment:
If you don't want to make effect title editable then remove this method.

Reply:
This method is required by scripted effects so that their `title` property is writable.